docs/governance-and-automation/lifecycle-management/inactive-content.md

```diff
@@ -32,12 +29,6 @@ The **Viva Engage activity** collection is also accomplished through the Microso
 Activity is being monitored from the Syskit Point installation onward!** This applies to SharePoint and Exchange workloads since historical data for these workloads is not available in usage reports.
 :::
 
-## Define What Is Inactive
```

Comment:
ovdje se može linkati na IW admin policy article
i dalje se može definirati što je inactive sa IW policyima, samo što se jedan ili više IW policya može kroz rule ili ručno applyati na željene workspaceve